libpod/container_inspect.go

```diff
@@ -174,6 +174,9 @@ type InspectContainerConfig struct {
 	StopSignal uint `json:"StopSignal"`
 	// Configured healthcheck for the container
 	Healthcheck *manifest.Schema2HealthConfig `json:"Healthcheck,omitempty"`
+	// OSArguments are the full command plus arguments of the process the
+	// container has been created with.
+	OSArguments []string `json:"OSArguments"`
```

Comment:
Make this `,omitempty` so we don't include it if the frontend for libpod doesn't include it.